app/views/candidate_interface/application_form/review_submitted.html.erb

```diff
@@ -1,5 +1,10 @@
 <% content_for :title, t('page_titles.submitted_application') %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application dashboard') %>
+
+<% if @review_previous_application %>
```

Comment:
The pages are identical with the exception of the content of the back link. Perhaps the rest of the page could go into a partial to avoid this?

Reply:
Not sure I understand what you mean by pages being identical. I can't see anything too bad about this apart from the `content_for` call being (almost) duplicated. I can't think of a neat way to avoid that. You could specify the default parameter value as a constant in the `ViewHelper` so that you could do:

```
<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, @review_previous_application ? 'Back to application dashboard') : ViewHelper::DEFAULT_BACK_LABEL %>
```

 But I think that is less clear.

Reply:
Sorry, that is misleading. Two actions render the same view but need different text in the back link. I think the current solution is probably fine then 👍